hack/generate_config_updater_kubeconfig.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+CLUSTER="$1"
+
+if [[ -z $CLUSTER ]]; then
+  echo "usage: $0 clusterName"
+  exit 1
+fi
+
+export VAULT_ADDR=https://vault.ci.openshift.org
+if ! vault kv get kv/dptp/build_farm >/dev/null; then
+  echo "Must be logged into vault, run 'vault login -method=oidc'"
+  exit 1
+fi
+
+oc config use-context "$CLUSTER"
```

Comment:
Can we use `--context` in individual calls to avoid side effects for whoever runs this?

Reply:
This line would fail (and the whole script exit with failure) if the context $CLUSTER is not found.
Would it ensure no side-effects out of running this?